rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -147,3 +151,104 @@ def find_exactly_one_fluid_loop(rmi, loop_id):
         loop_id,
         rmi,
     )
+
+
+def has_heating_system(rmi_b, hvac_b_id):
+    """
+    Check whether heating system exists or not.
+
+    Parameters
+    ----------
+    rmi_b json
+        To evaluate if the hvac system is modeled as Sys-2 in the B_RMI.
+
+    hvac_b_id list
+        The id of the hvac system to evaluate.
+
+    Returns
+    -------
+    If heating system exists, it returns true. Otherwise, it returns false.
+    """
+    heating_system = find_exactly_one_hvac_system(rmi_b, hvac_b_id).get(
+        "heating_system"
+    )
+
+    return (
+        heating_system is not None
+        and heating_system.get("heating_system_type") is not None
+        and heating_system["heating_system_type"] != HEATING_SYSTEM.NONE
+    )
+
+
+def has_cooling_system(rmi_b, hvac_b_id):
```

Comment:
All my comments for `has_heating_system` also apply to `has_cooling_system`, `has_preheat_system`, and `has_fan_system`.